helper/install_devel_packages.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
```

Comment:
I know, this comment is beyond the scope of this PR and probably a "matter of taste", but... you asked for comments so it's free. :smile: 

Have you considered to check for a specific operating system than for a specific command? I would have something in my mind like investigating `/etc/os-release`. The reason for my idea is to make it possible to "fine-tune" it for a specific OS. Checking for a specific command would probably not enough.

Reply:
Well I thought about how complex the coding should be, but this is a helper and zypper is used on suse/SLES exclusively and dnf is found on Fedora/RHEL/CentOS. dnf can also be used on suse but that's a rare case and zypper will match first in this case. In the end I thought for a start and as a helper this is good enough. If we need to make this more complex and error proof it can be done on follow up requests

Reply:
`zypper` works on Fedora, and `dnf` works on openSUSE. So it might make sense to be a little discriminating here...

Reply:
:smile: ok so any nice way to detect the os from bash other than reading /etc/os-release ? I guess not. It sounds trivial but I think it's not: 

```
NAME
VERSION
ID
ID_LIKE
VERSION_ID
```

Looking for a simple solution...

Reply:
Take a look here:

* https://stackoverflow.com/questions/1977306/python-module-to-detect-linux-distro-version

and you know what I mean :) there is no safe way imho

Reply:
You can just source `/etc/os-release` and use `ID` and `ID_LIKE` to figure things out.

Reply:
ok let me collect this: I'm aware of:

* ID="sles"
* ID="opensuse-*"
* ID=fedora

@Conan-Kudo I'm lacking the information for CentOS and RHEL if dnf is used. Can you provide me that part ?

Thanks

Reply:
CentOS uses `centos`, RHEL uses `rhel`.

Reply:
@Conan-Kudo Thanks, I updated the script, can you take a look ?